recipes/molsystem/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "molsystem" %}
+{% set version = "2021.5.5" %}
+{% set sha256 = "0ecb5cedf29090258c19960f92486d1a9afe24ea27772f13e1ebc4e2be2f869f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/molsystem-{{ version }}.tar.gz
```

Comment:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Reply:
No problem, but curiosity: why just that one place and not the 3 previous places {{ name }} is used? It would seem like all or nothing would make sense.

Reply:
I think you're misinterpreting the suggestion? I'm suggesting that you be consistent with the 3 previous uses of `{{ name }}` so that the entire URL *is* all or nothing. 😉

Reply:
Ah! Yes, I read it backwards! I used Grayskull and it put it that way, so I left it as is. I'll make the change and thanks for pointing out my backwards reading. I don't use the web interface much (old emacs guy....)

Reply:
Uff. Sorry! But the nice "Commit suggestion" seemed to commit, but not to my branch. Is that as expected? As I mentioned, I am not familiar with this web interface....

Reply:
It *should* have been committed to the remote branch (on GitHub). Regardless, it seems to be here now.